src/modules/roc_audio/resampler_reader.h

```diff
@@ -64,12 +66,12 @@ class ResamplerReader : public IReader, public core::NonCopyable<> {
 private:
     bool init_frames_(core::BufferPool<sample_t>&);
     void renew_frames_();
+    const size_t frame_size_;
 
     Resampler resampler_;
     IReader& reader_;
 
     core::Slice<sample_t> frames_[3];
-    const size_t frame_size_;
     bool frames_empty_;
```

Comment:
This frame* fields were grouped together, I suggest to keep this as is. The same is true for some other classes.

Reply:
done.